build/Cortex-M3_MPS2_QEMU_GCC/target-specific-source/startup.c

```diff
@@ -92,26 +140,23 @@ void prvGetRegistersFromStack( uint32_t *pulFaultStackAddress )
     for( ;; );
 }
 
-static void Default_Handler( void ) __attribute__( ( naked ) );
-void Default_Handler(void)
+
+void Default_Handler( void )
 {
     __asm volatile
     (
-        "Default_Handler: \n"
-        "    ldr r3, NVIC_INT_CTRL_CONST  \n"
-        "    ldr r2, [r3, #0]\n"
-        "    uxtb r2, r2\n"
-        "Infinite_Loop:\n"
-        "    b  Infinite_Loop\n"
-        ".size  Default_Handler, .-Default_Handler\n"
-        ".align 4\n"
-        "NVIC_INT_CTRL_CONST: .word 0xe000ed04\n"
+        " ldr r3, NVIC_INT_CTRL_CONST            \n" /* Load the address of the interrupt control register into r3. */
+        " ldr r2, [r3, #0]                       \n" /* Load the value of the interrupt control register into r2. */
+        " uxtb r2, r2                            \n" /* The interrupt number is in the least significant byte - clear all other bits. */
+        "Infinite_Loop:                          \n" /* Sit in an infinite loop - the number of the executing interrupt is held in r2. */
+        " b  Infinite_Loop                       \n"
```

Comment:
The change removes the ".size Default_Handler, .-Default_Handler\n" line. Is it intentional?